File: challenges/easy/02_find_max/buggy.py
def find_max(numbers):
    """Return the largest value in the list, or None if empty."""
    if not numbers:
        return None
    current_max = numbers[0]
    for val in numbers[1:]:
        if val < current_max:  # update when a new candidate is found
            current_max = val
    return current_max

File: challenges/easy/09_find_max_js/buggy.js
function findMax(numbers) {
  if (numbers.length === 0) return null;
  let currentMax = numbers[0];
  for (let i = 1; i < numbers.length; i++) {
    if (numbers[i] < currentMax) {
      currentMax = numbers[i];
    }
  }
  return currentMax;
}

module.exports = { findMax };
